src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/ampMetadataExtractor/index.js

```diff
@@ -1,17 +1,28 @@
 import Url from 'url-parse';
+import nodeLogger from '#lib/logger.node';
+import { INCLUDE_IFRAME_REQUEST_RECEIVED } from '#lib/logger.const';
+
+const logger = nodeLogger(__filename);
 
 /**
  * Returns an object containing metadata extracted from an Include url
  *
  * @param {string} href The href value from an Include Block.
  */
 
-const ampMetadataExtractor = (href, baseUrl) => {
+const ampMetadataExtractor = (href, baseUrl, classification) => {
+  if (classification !== 'vj-supports-amp') return null;
```

Comment:
I think introducing the concept of classifications couples `ampMetadataExtractor` to the `includeClassifier` uneccessarily. I think this perhaps should stay in the `index.js` as this is the 'orchestration' component that knows about all the different kinds of includes.